lib/src/util.dart

```diff
@@ -435,6 +435,29 @@ extension TypeOrBound on DartType {
   }
 }
 
+extension IsA on DartType {
```

Comment:
Not sure this is the best way to check this, but it seems to work, and be useful. I even wrote a test for it, since it's going into the more general codemod infrastructure.

Reply:
We have this existing extension which does something very similar: https://github.com/Workiva/over_react_codemod/blob/master/lib/src/util/element_type_helpers.dart#L26

so it might make sense to consolidate them.

Reply:
Oh, yes. I'd seen that, but didn't realize that it actually did what I wanted. Switched to using it, and removed the extension. And now I don't need the test.